executor/src/witgen/fixed_evaluator.rs

```diff
@@ -5,12 +5,12 @@ use powdr_ast::analyzed::AlgebraicReference;
 use powdr_number::FieldElement;
 
 /// Evaluates only fixed columns on a specific row.
-pub struct FixedEvaluator<'a, T> {
+pub struct FixedEvaluator<'a, T: FieldElement> {
```

Comment:
The usual practice in rust is no have those only for the impls, but it's also fine like this if it would be too much work to remove it from GlobalConstraints